juju/juju
# Refactor shared interfaces.

## Description of change

Move Unit, Address and Charm interfaces from the containerizer package
to the provisioner package.  Unit is no longer required in the containerizer,
as of PR 10872.

## QA steps

This is a refactoring of interfaces.  No change to functionality should occur.

1. juju bootstrap lxd
2. juju deploy ubuntu --to lxd
